.github/workflows/wheel.yml

```diff
@@ -0,0 +1,216 @@
+# this is based on the wheel workflow in GalSim, but adapted to our needs
+name: build wheels and sdist
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: 'The git ref to build wheels for. This will trigger a pypi upload.'
+        default: ''
+        required: false
+        type: string
+      cibw_skip:
+        description: 'Python versions to skip when building wheels.'
+        default: 'cp36* cp37* pp* cp38*'
+        required: false
+        type: string
+  pull_request: null
+  release:
+    types:
+      - published
+
+concurrency:
+  group: pypi
+  cancel-in-progress: false
+
+env:
+  PYVER: '3.11'
+  CIBW_SKIP_VAL: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.cibw_skip || 'cp36* cp37* pp* cp38*' }}
+
+jobs:
+  linux-manylinux:
+    name: linux-manylinux
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.ref || '' }}
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '${{ env.PYVER }}'
+
+      - name: build wheels
+        uses: pypa/cibuildwheel@v2.21.1
+        env:
+          CIBW_BUILD: "*manylinux*"
+          CIBW_ARCHS: auto64
+          CIBW_SKIP: ${{ env.CIBW_SKIP_VAL }}
+          # I think yum might always work here.  But leave all options available.
+          CIBW_BEFORE_ALL: yum install -y bzip2-devel || apt-get install libbz2-dev || apk add --upgrade bzip2-dev
```

Comment:
yum on ubuntu?

Reply:
It tries each command in turn. Agreed this is overkill and we only need apt